frontend/src/features/books/EditBookForm.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { useParams, useNavigate } from 'react-router-dom'
+import { useEffect, useState } from 'react'
+import { API_URL } from '../../constants'
+
+function EditBookForm() {
+    const [book, setBook] = useState(null)
+    const [title, setTitle] = useState('')
+    const [author, setAuthor] = useState('')
+    const [description, setDescription] = useState('')
+    const { id } = useParams()
```

Comment:
@nshelokar-vf, did we handle the case where `id` is not present?

Reply:
Previously not, but now I am handling it.